fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTray.kt

```diff
@@ -92,8 +95,10 @@ fun TabsTray(
     val pagerState = rememberPagerState(initialPage = 0)
     val scope = rememberCoroutineScope()
     val isInMultiSelectMode = multiselectMode is TabsTrayState.Mode.Select
+    val normalTabCountTotal = normalTabs.size + inactiveTabs.size
 
-    val animateScrollToPage: ((Page) -> Unit) = { page ->
+    val onTabPageIndicatorClicked: ((Page) -> Unit) = { page ->
+        onTabPageClick(page)
         scope.launch {
             pagerState.animateScrollToPage(page.ordinal)
         }
```

Comment:
this function will get re-tooled as part of [Bug 1819194](https://bugzilla.mozilla.org/show_bug.cgi?id=1819194)